README.md

```diff
@@ -65,6 +69,13 @@ to publish events on your behalf.
 
 [Create an event](http://localhost:8000/admin/census/event/add/).
 
+### Setup Maps API
+In our event creation form we have the google autocomplete api available.
+
+1. Enable the [Maps Javascript API](https://console.cloud.google.com/apis/library/maps-backend.googleapis.com)
+1. Create an API Key: https://console.cloud.google.com/google/maps-apis/apis/maps-backend.googleapis.com/credentials?
```

Comment:
There is a ? at the end of this sentance.  Also I think some more details are needed here.